kingpin/actors/aws/iam.py

```diff
@@ -27,6 +27,7 @@
 from kingpin.actors.aws import base
 from kingpin.actors import exceptions
 from kingpin.constants import REQUIRED
+from kingpin.constants import STATE
```

Comment:
I took a stab at refactoring this so its more powerful down the road.. the new mechanism will make it easy for individual actors to create complex option validation methods without cluttering up their individual actor classes.